### PR TITLE
DFC-551: Rename one-login-language-toggle to frontend-language-toggle

### PR DESCRIPTION
**Pull request by di-aholme**
### Description

Renames the OneLogin Language Toggle to follow a compliant naming convention

Version bump to major version 1.0.0

### Tickets

(DFC-551)[https://govukverify.atlassian.net/browse/DFC-551]
(DFC-560)[https://govukverify.atlassian.net/browse/DFC-560]

### CODE REVIEW

File: src/components/frontend-language-toggle/README.md [hunk 1]
```diff
@@ -58,28 +58,28 @@ The package is owned by the DI Frontend Capability team, part of the development
 
 1. Install NPM package
    ```sh
-   npm install one-login-language-toggle
+   npm install language-toggle
```
Comment (Joanneyoung01) [inline src/components/frontend-language-toggle/README.md:61]: i think we need to keep 

when i run 
`npm install language-toggle` it says not found so i think we need to keep the org name

File: src/components/frontend-language-toggle/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    ] /* Specify library files to be included in the compilation. */,
+    // "allowJs": true,                       /* Allow javascript files to be compiled. */
+    // "checkJs": true,                       /* Report errors in .js files. */
+    // "jsx": "react",                           /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */
+    "declaration": true /* Generates corresponding '.d.ts' file. */,
+    "declarationMap": true /* Generates a sourcemap for each corresponding '.d.ts' file. */,
+    "sourceMap": true /* Generates corresponding '.map' file. */,
+    // "outFile": "./",                       /* Concatenate and emit output to single file. */
+    "outDir": "./dist" /* Redirect output structure to the directory. */,
+    // "rootDir": "./src" /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */,
```
Comment (Joanneyoung01) [inline src/components/frontend-language-toggle/tsconfig.json:19]: Do we need to keep the comments in?

Comment (di-aholme) [reply src/components/frontend-language-toggle/tsconfig.json]: Updated and removed the unused config

File: src/config/nunjucks.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 const nunjucks = require("nunjucks");
 const i18next = require("i18next");
-const addLanguageParam = require("@govuk-one-login/one-login-language-toggle/language-param-setter");
+const addLanguageParam = require("@govuk-one-login/frontend-language-toggle/cjs/index.cjs");
```
Comment (Joanneyoung01) [inline src/config/nunjucks.js:3]: Did you mention that this is tested using the commonJS method I can see you’ve imported the below line using a const/require const addLanguageParam = require("@govuk-one-login/frontend-language-toggle/cjs/index.cjs"); in the src/config/nunjucks.js file, should this line be replaced in the screenshot attached? 
[@Andy Holme](https://gds.slack.com/team/U060YSALWP6)
 (edited)

Comment (Joanneyoung01) [reply src/config/nunjucks.js:3]: const addLanguageParam = require("@govuk-one-login/one-login-language-toggle");

File: src/components/frontend-language-toggle/README.md [hunk 4]
```diff
@@ -107,35 +107,35 @@ The package is owned by the DI Frontend Capability team, part of the development
...
+       next();
+     }
+   });
    ```
 
 7. Import the addLanguageParam function into your Nunjucks configuration file (e.g. nunjucks.js) and make it accessible to Nunjucks views. It is used by the component to set the URL.
 
    ```js
-   const addLanguageParam = require("@govuk-one-login/one-login-language-toggle/language-param-setter");
+   const addLanguageParam = require("@govuk-one-login/frontend-language-toggle/language-param-setter");
```
Comment (SophiaDWokoma) [inline src/components/frontend-language-toggle/README.md:134]: We should also probably add the import version too.
`import addLanguageParam from "@govuk-one-login/frontend-language-toggle";`

**Review by SophiaDWokoma — CHANGES_REQUESTED** [review]

**Review by Joanneyoung01 — APPROVED** [review]

**Review by SophiaDWokoma — APPROVED** [review]

**Pull request merged**